backend/src/api/users/users.api.ts

```diff
@@ -61,10 +61,10 @@ const initUsersApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
     ) {
       const { id } = req.params;
 
-      const isDeleted = await userService.delete(Number(id));
+      const isDeleted = await userService.delete(req.user.id, Number(id));
 
       return rep
-        .status(isDeleted ? HttpCode.OK : HttpCode.NOT_FOUND)
+        .status(isDeleted ? HttpCode.NO_CONTENT : HttpCode.NOT_FOUND)
```

Comment:
Why do you use NO_CONTENT status instead of OK?

Reply:
@v0ldemar01  We don't send entity

Reply:
@v0ldemar01 in the documentation it was written about 204 and 404

Reply:
I think it makes sense to return 204 on a successful delete request, which has no body to return

Reply:
@kalinkaaaa14 okey, leave your version. There was a misunderstanding with deleting users

Reply:
ok, understood